build.sbt

```diff
@@ -1,12 +1,14 @@
+organization := "gatling-mqtt"
```

Comment:
You don't have to set the project organization to use this plugin. However, if you want to set, how about the project organization "com.github.mnogu"? The sbt documentation says that the project organization is a reverse domain name, though I don't own "github.com".

> By convention, this is a reverse domain name that you own, typically one specific to your project. It is used as a namespace for projects. 
http://www.scala-sbt.org/0.13/docs/Howto-Project-Metadata.html#Set+the+project+organization